utils/coreutils/utils.go

```diff
@@ -19,6 +20,8 @@ import (
 
 const (
 	GettingStartedGuideUrl = "https://github.com/jfrog/jfrog-cli/blob/v2/guides/getting-started-with-jfrog-using-the-cli.md"
+	Artifactory            = "Artifactory"
+	Xray                   = "Xray"
 )
 
```

Comment:
Let's use a dedicated const type for each of one the products. For example:
```suggestion
)

type MinVersionProduct string

const (
	Artifactory  MinVersionProduct = "Artifactory"
	Xray         MinVersionProduct = "Xray"
	DataTransfer MinVersionProduct = "Data Transfer"
	DockerApi    MinVersionProduct = "Docker API"
)
```

And then change ValidateMinimumVersion to this:

```go
func ValidateMinimumVersion(product MinVersionProduct, currentVersion, minimumVersion string) error {
```